### ISSUE TITLE
Helpdesk #172398 - dr14477 - NSI Koala Survey

### ISSUE THREAD

**Issue by rosemaryjoconnor** [AtlasOfLivingAustralia/data-management#881]
Additional data to be added to current dataset

**Comment by rosemaryjoconnor** [comment]
New dataset updated and merged with existing dataset. Waiting on databox for load.

**Comment by rosemaryjoconnor** [comment]
Duplicate of #880 so will close

**Comment by rosemaryjoconnor** [comment]
dr14477 successfully loaded in databox

**Comment by rosemaryjoconnor** [comment]
dr14477- uploaded to Prod collectory. Preingestion successful. Waiting on index run overnight
